todo-frontend/src/components/Edit.vue

```diff
@@ -0,0 +1,66 @@
+<template>
```

Comment:
Edit i Create su isti, imaju identicnu formu, identicno izgledaju (koliko vidim iz koda)
jedino sto se razlikuje je sto jedan poziva edit drugi add todo
to bi mogla da bude jedna ista komponenta, a na click tog submit buttona da emituje neki event "submit" npr, a onda tu gde se koristi ta forma da se hendla da li je edit ili create

u sustini imao bi EditTodo i CreateTodo komponente koje hendlaju sta se desava na neki event, ali i TodoForm u kojoj bi bila sama forma za todo.